T/Torch/Torch/build_tarballs.jl

```diff
@@ -54,8 +58,6 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
```

Comment:
We'll eventually require `dependencies` to be a vector of `AbstractDependency`s
```suggestion
dependencies = Dependency[
```

Reply:
Done